source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,218 @@
+
+This procedure assumes familiarity with the referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.
+
+.. _minio-k8s-deploy-operator-kustomize-repo:
+
+Install the MinIO Operator using Kustomize
+------------------------------------------
+
+The following procedure uses ``kubectl --kustomize`` to install the Operator from the MinIO Operator GitHub repository:
+
+.. important::
+
+   Do not use ``kubectl krew`` or similar methods to update or manage the MinIO Operator installation.
+   If you use Kustomize to install the Operator, you must use Kustomize to manage that installation.
+
+#. Install the latest version of Operator
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      kubectl apply -k github.com/minio/operator\?ref=|operator-version-stable|
+
+   The output resembles the following:
+
+   .. code-block:: shell
+
+      namespace/minio-operator created
+      customresourcedefinition.apiextensions.k8s.io/miniojobs.job.min.io created
+      customresourcedefinition.apiextensions.k8s.io/policybindings.sts.min.io created
+      customresourcedefinition.apiextensions.k8s.io/tenants.minio.min.io created
+      serviceaccount/console-sa created
+      serviceaccount/minio-operator created
+      clusterrole.rbac.authorization.k8s.io/console-sa-role created
+      clusterrole.rbac.authorization.k8s.io/minio-operator-role created
+      clusterrolebinding.rbac.authorization.k8s.io/console-sa-binding created
+      clusterrolebinding.rbac.authorization.k8s.io/minio-operator-binding created
+      configmap/console-env created
+      secret/console-sa-secret created
+      service/console created
+      service/operator created
+      service/sts created
+      deployment.apps/console created
+      deployment.apps/minio-operator created
+
+   Verify the Operator pods are running:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get pods -n minio-operator
+
+   The output resembles the following:
+
+   .. code-block:: shell
+
+      NAME                              READY   STATUS    RESTARTS   AGE
+      console-6b6cf8946c-9cj25          1/1     Running   0          99s
+      minio-operator-69fd675557-lsrqg   1/1     Running   0          99s
+
+   In this example, the ``minio-operator`` pod is MinIO Operator and the ``console`` pod is the Operator Console.
+
+   You can modify your Operator deplyoment by applying Kustomize patches.
+   You can find examples for common configurations in the `Operator GitHub repository <https://github.com/minio/operator/tree/master/examples/kustomization>`__.
+
+#. (Optional) Configure access to the Operator Console port
```

Comment:
```suggestion
#. *(Optional)* Configure access to the Operator Console port
```

Reply:
oh yes and changed the Helm page to match